src/library_webgl.js

```diff
@@ -1784,35 +1798,50 @@ var LibraryGL = {
       GL.timerQueriesEXT[id] = null;
     }
   },
+#endif
 
+#if !USE_WEBGL2 // WebGL2 alias defined in library_webgl2.js
   glIsQueryEXT__sig: 'ii',
   glIsQueryEXT: function(id) {
     var query = GL.timerQueriesEXT[id];
     if (!query) return 0;
     return GLctx.disjointTimerQueryExt['isQueryEXT'](query);
   },
+#endif
 
+#if !USE_WEBGL2 // WebGL2 alias defined in library_webgl2.js
   glBeginQueryEXT__sig: 'vii',
   glBeginQueryEXT: function(target, id) {
 #if GL_ASSERTIONS
     GL.validateGLObjectID(GL.timerQueriesEXT, id, 'glBeginQueryEXT', 'id');
 #endif
     GLctx.disjointTimerQueryExt['beginQueryEXT'](target, GL.timerQueriesEXT[id]);
   },
+#endif
 
+#if !USE_WEBGL2 // WebGL2 alias defined in library_webgl2.js
   glEndQueryEXT__sig: 'vi',
   glEndQueryEXT: function(target) {
     GLctx.disjointTimerQueryExt['endQueryEXT'](target);
   },
+#endif
```

Comment:
Generally we have reused the `#if !USE_WEBGL2` blocks across multiple functions, so can just have this once for all of these.

Reply:
Done -- actually I removed most of the preprocessor blocks now.